### ISSUE TITLE
Data transfer calculator

### ISSUE THREAD

**Issue by auanasgheps** [ferraridamiano/ConverterNOW#35]
Ciao,
Greetings from Italy.
Thanks for this unit converter, until today I had to use shady apps filled with ads.

There's a calculation I do a lot and I think would fit nice the app: data transfer calculator.
E.g: How much time does it take to transfer 2.5 GB at 3.8 MB/s?
It should allow the user to pick whatever unit combination desires.
I use [this one](http://www.calctool.org/CALC/prof/computing/transfer_time)  but is ugly and not mobile friendly

Thanks!

**Comment by ferraridamiano** [comment]
Ciao @auanasgheps! Thanks for your contribution!
Some month ago I started to explore some intuitive design for a new tool section and I started with that conversion. Now that branch is paused because I'm refactoring the app and it will take some time. 
I will close this issue for now because I first need to setup all the tools structure as mentioned here #6
Don't worry I will consider your suggestion in the future, it's not just the priority as of now.

Saluti e grazie ancora per la tua proposta! 👍
